### PR TITLE
Add My reservation

### LINKED ISSUE #10: [3pt] My reservations Page
When the user clicks the "My reservations" link in the navigation panel they can see a list of their reservations (with information about item name, date and city).
- Add Style
#### Hint:
- GET Request...

### PR DESCRIPTION
**Pull request by Cabraham1**
image
image

### CODE REVIEW

File: app/javascript/components/MyReservations.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        showError('Request failed!', error);
+      }
+    };
+    allReservation();
+  }, [username, dispatch]);
+
+  const handleDeleteReservation = () => {
+    // Handle delete logic here
+  };
+
```
Comment (edahigure) [inline app/javascript/components/MyReservations.js:45]: Why is this method not implemented?

File: app/javascript/components/MyReservations.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  return (
+    <div className="flex flex-col justify-center items-center w-full bg-slate-300">
+      <h2 className="text-center text-3xl font-bold py-[5rem]">
+        Reservation List
+      </h2>
+      {loading ? (
+        <div className="flex justify-center items-center h-[100%] w-full">
```
Comment (edahigure) [inline app/javascript/components/MyReservations.js:52]: When I make the nav, I have some problems with the containers and its components, like when I make the screen smaller the components inside don't change its size. Is it possible to wrap your pages in a container, and that all its components shrink together?  Also if the style may be more similar to the design given :

**Review by edahigure — DISMISSED** [review]
# Changes required ♻︎

Hi @Cabraham1

You have done a great work!. Just I think the part is not complete as to make a merge. I would suggest to finish the method for destroy the reservation. And see if the style may be more like the design given. 
# Highlights:


- Great use of the Api fetch functions ✅
- Lectures slice implemented  ✅
- List all reservations page implemented ✅
🚀

File: app/javascript/components/MyReservations.js (hunk 1 above)
Comment (Mar12358) [reply app/javascript/components/MyReservations.js:45]: @edahigure I need to add the "removed" field into the tables both for reservations and lectures, let's merge this and finish the "delete" implementation after updating the API remove feature

File: app/javascript/components/MyReservations.js (hunk 2 above)
Comment (Mar12358) [reply app/javascript/components/MyReservations.js:52]: @edahigure, please ask @Cabraham1 privately for assistance on this, so that we close the PR

**Review by Mar12358 — APPROVED** [review]
Hi @Cabraham1 , it works as expected.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- The user is able to create a reservation and see a list of them after it.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**